### DISCUSSION TITLE
Fedora Onyx'e yükleme sorunu

### DISCUSSION THREAD

**Discussion by dsg-sudo** [realsuayip/django-sozluk#86] [Q&A]
Merhaba,

Linux ve yazılım konularında yeniyim. Bilgisayarda Fedora.39 Onyx yüklü. Projeyi kurmaya çalıştım ancak aşağıda görüldüğü üzere containerlar oluşsa da kullanıcı adı-şifre oluşturma ve server'ı çalıştırma aşamalarına geçemedim. Nerede hata yapıyorum ya da Fedora Onyx'e özel, dikkat edilmesi gereken bir durum var mı?

Yanıt için teşekkürler.

![Snapshot_2023-11-10_13-48-19](https://github.com/realsuayip/django-sozluk/assets/110290794/0e0711d6-2159-4aa4-833c-2cc9f128e70c)

![Snapshot_2023-11-10_14-18-56](https://github.com/realsuayip/django-sozluk/assets/110290794/a64b5b28-d29d-4171-87af-a26159c2f582)

![Snapshot_2023-11-10_14-21-01](https://github.com/realsuayip/django-sozluk/assets/110290794/71ba3f10-41a2-4973-8046-cc11f03260c1)

**Comment by realsuayip** [comment]
selamlar. Docker ile çalıştırdığınız için sistem python'unu kullanamazsınız. `sudo docker ps` diyip container'ler ne durumda diye bir bakın. her şey yolundaysa `127.0.0.1:8000` adresinde olmalı sözlük

**Comment by dsg-sudo** [comment]
Containerlar şu şekilde:

![Snapshot_2023-11-10_19-10-20](https://github.com/realsuayip/django-sozluk/assets/110290794/b92e8299-31f1-432f-b65c-2144b02d0cc3)
